app/_src/networking/transparent-proxy/reachable-backends.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Reachable Backends
+content_type: how-to
+---
+
+{% assign docs = "/docs/" | append: page.release %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign tproxy = site.data.tproxy %}
+
+{% warning %}
+This feature works only when [MeshService]({{ docs }}/networking/meshservice) is enabled.
+{% endwarning %}
+
+Reachable Backends provides similar functionality to [Reachable Services]({{ docs }}/networking/transparent-proxy/reachable-services/), but it applies to resources such as [MeshService]({{ docs }}/networking/meshservice), [MeshExternalService]({{ docs }}/networking/meshexternalservice), and [MeshMultiZoneService]({{ docs }}/networking/meshmultizoneservice).
```

Comment:
I think here your doc is biased towards warnings in the newer stuff. I think it should be the other way round or 2 way